schemas/sample_registration.json

```diff
@@ -0,0 +1,126 @@
+{
+  "name": "sample_registration",
+  "description": "Collection of elements required to register the required Donor-Specimen-Sample data to the ARGO Platform. Registration of samples is required before genomic and clinical data submission can proceed.",
+  "fields": [
+    {
+      "name": "program_id",
+      "valueType": "string",
+      "description": "Unique identifier of the ARGO program.",
+      "restrictions": {
+        "required": true
+      }
+    },
+    {
+      "name": "submitter_donor_id",
+      "valueType": "string",
+      "description": "Unique identifier of the donor, assigned by the data provider.",
+      "restrictions": {
+        "required": true,
+        "regex": "[A-Za-z0-9\\-\\._]{1,64}"
```

Comment:
don't have very strong opinion on this, but do we expect IDs only start with `[A-Za-z0-9]`, ie, can't start with `.`, `-`, or `_`? If yes, the regex needs to be updated to enforce that rule.

Reply:
let's go with that then